app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: list) -> None:
+    for player in elves:
+        player.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list) -> None:
```

Comment:
Please specify correct typing
```suggestion
def feast_of_the_dwarves(dwarves: list[Dwarf]]) -> None:
```